dynamiq/nodes/llms/base.py

```diff
@@ -140,6 +140,13 @@ def get_context_for_input_schema(self) -> dict:
         """Provides context for input schema that is required for proper validation."""
         return {"instance_prompt": self.prompt}
 
+    def format_messages_for_provider(self, messages: list[dict]) -> list[dict]:
```

Comment:
Let's maybe call this `def get_messages` and move 
`return prompt.format_messages(**dict(input_data))` here as default?
For Mistral also make extra formatting